src/patternfly/components/FileUpload/examples/FileUpload.md

```diff
@@ -0,0 +1,150 @@
+---
+title: File upload
+section: beta
+cssPrefix: pf-c-file-upload
+---
+
+## Examples
+
+```hbs title=Basic-file-upload
+{{#> file-upload file-upload--id="basic-file-upload"}}
+  {{#> file-upload-input-group}}
+    {{#> input-group}}
+      {{> file-upload-text-input
+        file-upload-text-input--aria-label="Upload a file"
+        file-upload-text-input--attribute='placeholder="Drag a file here or browse to upload"'
+        }}
+      {{#> button button--modifier="pf-m-control"}}
+        Browse...
+      {{/button}}
+      {{#> button button--modifier="pf-m-control" button--attribute="disabled"}}
+        Clear
+      {{/button}}
+    {{/input-group}}
+  {{/file-upload-input-group}}
+  {{#> file-upload-textarea file-upload-textarea--aria-label="Text area empty"}}
```

Comment:
"Text area empty" is really awkward to me; would "Text area is empty" or "Empty text area" be better?